frontend/src/metabase/home/components/NewUserOnboardingModal.jsx

```diff
@@ -1,97 +1,115 @@
-
-                            <p className="clearfix"><img className="float-left mr2" width="40" height="40" src="app/assets/img/onboarding_illustration_dashboards.png" />You (and anyone on your team) can save answers in Dashboards, so you can check them often. It's a great way to quickly see a snapshot of your business.</p>
-                        </div>
-                        <div className="px4 py2 text-grey-2 flex align-center">
-                            {this.renderStep()}
-                            <button className="Button Button--primary flex-align-right" onClick={() => (this.nextStep())}>Continue</button>
-                        </div>
-                    </div>
-                :
-                    <div className="bordered rounded shadowed">
-                        <div className="pl4 pr4 pt4 pb1 border-bottom">
-                            <h2>Let's try asking a question!</h2>
-
-                            <p>We'll take a quick look at the Query Builder, the main tool you'll use in Metabase to ask questions.</p>
-                        </div>
-                        <div className="px4 py2 text-grey-2 flex align-center">
-                            {this.renderStep()}
-                            <span className="flex-align-right">
-                                <a className="text-underline-hover cursor-pointer mr3" onClick={() => (this.closeModal())}>skip for now</a>
-                                <Link to={Urls.question(null, "?tutorial")} className="Button Button--primary">Let's do it!</Link>
-                            </span>
+        return (
+            <div style={{ maxHeight: '100%', transition: 'height 300ms linear' }}>
+                <OnboardingImages
+                    currentStep={currentStep}
+                />
+                <div className="p4 pb3 text-centered">
+                    <h2>{currentStep.title}</h2>
+                    <p className="ml-auto mr-auto text-paragraph" style={{ maxWidth: 420 }}>
+                        {currentStep.text}
+                    </p>
+                    <div className="flex align-center py2 relative">
+                        <div className="ml-auto mr-auto">
+                            <StepIndicators
+                                currentStep={step}
+                                steps={STEPS}
+                                goToStep={step => this.setState({ step })}
+                            />
                         </div>
+                        <a
+                            className="link flex-align-right text-bold absolute right"
+                            onClick={() => (this.nextStep())}
+                        >
+                            Next
```

Comment:
Should we have a different link text for the last step that indicates that the welcome tutorial will end?